### ISSUE TITLE
Help - survival recipes (ie trapdoor)?

### ISSUE THREAD

**Issue by lolkje** [XFactHD/FramedBlocks#75]
In Vanilla survival mode, a 2x3 grid of stone gives you walls and a 2x3 grid of planks gives you trapdoors.

With this mod, a 2x3 grid of framed blocks is always giving me framed walls. What am I not figuring out for how to build framed trapdoors in survival?

**Comment by XFactHD** [comment]
It appears that I fucked up the recipes and no one noticed until now.
The recipes for Framed Walls and Framed Trapdoors is identical for the exact reason you mentioned: in vanilla they have the same recipe layout).
The question is: how to fix? One idea would be two Framed Fences in the middle and two Framed Blocks on either side. Would that make sense or do you have a better idea?

**Comment by lolkje** [comment]
Two framed fences in the middle sounds reasonable for walls.

Thank you!

**Comment by XFactHD** [comment]
Decided to take another look after pushing those commits.
Turns out I had already accounted for this: The Framed Trapdoor is crafted with six Framed Slabs instead of six Framed Blocks.
To account for the lower material cost, you only get one trapdoor instead of two.

Small recommendation: Install JEI for easier access to the recipes, there are multiple recipes in FramedBlocks that may be a bit unexpected.
